### ISSUE TITLE
[A11Y] - Reader View | colors lack sufficient contrast ratio

### ISSUE THREAD

**Issue by Tribunal33** [pkp/pkp-lib#12661]
## Issue Description
There are several instances where there is a light grey color that has a transparency value below 1 that makes it difficult to read.   Additionally some links with hover colors are below 4.5:1 contrast ratio. The scope of this ticket will only cover the reader view areas.  
- [ ] Side navigation components on Book landing page. (Published, Series, Catagories, or subheadings in Publication Formats side nav elements, block plugins)
- [ ] Registration page (affects login link, Privacy statement link, breadcrumb link).  Current color is 3.81 ratio

## How to reproduce / Current behavior
Navigate to the Book landing page with the various plugins and settings for block and side navigation elements.  Also comes from the Publication Formats of any particular book.  

## What application are you using?
OMP 3.5.0-4 (but applicable for OJS/OPS).

## Stack used 
- Device: Laptop
- OS: Windows 11
- Browser: Chrome
- Version: Latest

## Testing method
Manual, using Screenreader

## Action
- Remove or decrease the transparency in the CSS or modify the color to be more visible. 

<img width="661" height="144" alt="Image" src="https://github.com/user-attachments/assets/4239aeff-4c72-4f56-b9b8-593c18fb390c" />

Expected contrast is 4.5:1 for AA and 7:1 for AAA and currently sits at 4.58:1 so this is theorically acceptable for AA but could decrease transparency to .66 and be AAA acceptable with a contrast ratio of 7.25 .  The lack of visibilitiy was suggested by a real user so there is a confirmation about it being difficult to see.   

<img width="231" height="57" alt="Image" src="https://github.com/user-attachments/assets/1b9365f4-e6dd-42a8-ae7d-f369253fb914" />

Here is the Login link, Privacy Statement Link, Breadcrumb "Home" link css selector with 3.81 contrast ratio

<img width="677" height="112" alt="Image" src="https://github.com/user-attachments/assets/28db805c-a719-41f8-b041-9314253eb85f" />

## Additional information
Side Nav components with light grey text.  WCAG 1.4.3 Level AA reference : https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum
 WCAG 1.4.6 Level AAA reference : https://www.w3.org/WAI/WCAG21/Understanding/contrast-enhanced
<img width="1677" height="339" alt="Image" src="https://github.com/user-attachments/assets/8346bbc7-a42e-40e8-9dfe-5d8e08528d94" />

OJS:
<img width="292" height="239" alt="Image" src="https://github.com/user-attachments/assets/834c6f1a-012b-4d72-af43-6b862c7c81fe" />

**Comment by Godoy0722** [comment]
Hello @israelcefrin pinging you here as I believe I covered all items for this issue accros OJS, OMP and OPS.
